staging/src/k8s.io/kubectl/pkg/util/i18n/translations/README.md

```diff
@@ -42,6 +42,10 @@ We use the English translation as the `msgid`.
 
 ## Regenerating the bindata file
 
+> Note: Regeneration of bindata is no more necessary for Kubernetes 1.22+ as
+> the translations are now embedded into the binary at compile time.
+> See: https://github.com/kubernetes/kubernetes/pull/99829
+
```

Comment:
@soltysh -- how about this note?

I couldn't think of convincing scenarios where translations to released Kubernetes versions will be updated, but still felt to keep the block and add the note.